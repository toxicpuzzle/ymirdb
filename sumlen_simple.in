SET a 999 -1 2 -999 10 1
SET b 11101111 111 2 0 1
SUM a 
SUM b
LEN a 
LEN b
bye