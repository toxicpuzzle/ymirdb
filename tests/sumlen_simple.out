> ok

> ok

> 12

> 11101225

> 6

> 5

> bye
